modules/gis/src/lib/reproject.d.ts

```diff
@@ -0,0 +1,7 @@
+import {BinaryGeometryData} from '../types';
+
+export function mapBinaryCoords(
```

Comment:
Would be nice to add JSDoc with human readable string (that shows up when hovering in vscode).